main/introduction.bs

```diff
@@ -18,6 +18,12 @@ with mechanisms for controlling who can access what.
 A <dfn export>Solid app</dfn> is an application
 that reads or writes data from one or more [=data pods=].
 
+A <dfn export>read operation</dfn> entails that information about a resource's existence or its description can be known. [[Source](https://github.com/solid/specification/issues/149#issue-568433265)]
+
+A <dfn export>write operation</dfn> entails that information about resources can be created or removed. [[Source](https://github.com/solid/specification/issues/126#issuecomment-569920473)]
```

Comment:
Wondering whether we need to explicitly mention delete here

Reply:
Perhaps. I've created an issue to track this question: https://github.com/solid/specification/issues/165 and some background on why it is not currently there. Unless the existing definitions are unclear with respect to "delete", can we revisit in future? We can move the issue forward in the meantime.

Reply:
Issue looks good - agree we can revisit in future.